Source/Scene/Scene.js

```diff
@@ -2548,6 +2550,8 @@ define([
     var scratchEyeTranslation = new Cartesian3();
 
     function render(scene, time) {
+        scene._pickPositionCache = {}; //clean cache
```

Comment:
Can we avoid doing this allocation per frame?  We are really careful about not doing allocation per vertex, per command, per primitive, etc., but we even try to avoid per frame allocations.

Instead, perhaps set a flag like `_pickPositionCacheDirty` to `true`.  Then in `pickPositionWorldCoordinates`, clear the cache and set dirty to false if it was true.